tools/Google.Cloud.Tools.DocUploader/Program.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using CommandLine;
+using Google.Cloud.Tools.DocUploader;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+
+
+Parser.Default.ParseArguments<CreateMetadataOptions, UploadFileOptions>(args)
+    .WithParsed<CreateMetadataOptions>(options => CreateMetadata(options))
+    .WithParsed<UploadFileOptions>(options => UploadFile(options));
+
+// Creates the Metadata object based on the specified options and then store it as a JSON file at options.Destination.
```

Comment:
This method does nothing to store data in `JSON` file.That decided by file format specified in `options.Destination` . may be we should update the comment accordingly.

Reply:
done